### PR TITLE
restore items

### LINKED ISSUE #230: Restore deleted items
Currently, deleted items cannot be restored again. This should be made possible

### PR DESCRIPTION
**Pull request by daniel17903**
Added a button to restore deleted items in CouchDB and in WooCommerce.

Closes #230

### CODE REVIEW

File: Frontend/src/components/TableEditors/Items/ItemPopupFormular.svelte [hunk 1]
```diff
@@ -49,15 +49,58 @@
...
+      ...(docIsDeleted
+        ? [
+            {
+              type: InputTypes.BUTTON,
+              group: "Wie­der­her­stel­len",
+              text: "Gelöschten Gegenstand wie­der­her­stel­len",
+              onClick: async () => {
+                if (
+                  confirm(
+                    "Soll dieser Gegenstand in der Datenbank und auf der Webseite wiederhergestellt werden?"
```
Comment (parttimenerd) [inline Frontend/src/components/TableEditors/Items/ItemPopupFormular.svelte:92]: Why mention the database here?

**Review by parttimenerd — CHANGES_REQUESTED** [review]

File: Frontend/src/components/TableEditors/Items/ItemPopupFormular.svelte (hunk 1 above)
Comment (daniel17903) [reply Frontend/src/components/TableEditors/Items/ItemPopupFormular.svelte:92]: My intention was to make clear that the item will be restored in the database (where it is currently marked as deleted) and in WooCommerce. Would you prefer another message?

Comment (skjerns) [reply Frontend/src/components/TableEditors/Items/ItemPopupFormular.svelte:92]: I think it's good to mention both!

**Comment by skjerns** [comment]
looks good :)

File: Frontend/src/components/TableEditors/Items/ItemPopupFormular.svelte (hunk 1 above)
Comment (parttimenerd) [reply Frontend/src/components/TableEditors/Items/ItemPopupFormular.svelte:92]: I would just mention that it is restored. Mentioning both confuses people that do not have that much insight into the inner workings of the application and none of the other messages seems to mention the database. Besides, "database" might be too generic.

**Pull request merged**